core/domain/email_manager.py

```diff
@@ -354,3 +356,53 @@ def send_role_notification_email(
     _send_email(
         recipient_id, feconf.SYSTEM_COMMITTER_ID,
         feconf.EMAIL_INTENT_EDITOR_ROLE_NOTIFICATION, email_subject, email_body)
+
+
+def send_feedback_message_email(recipient_id, feedback_messages):
+    """Sends an email when creator receives feedback message to an exploration.
+
+    Args:
+    - recipient_id: id of recipient user.
+    - feedback_messages: dictionary containing feedback messages.
+    """
+
+    email_subject = 'New messages on Oppia.'
+
+    email_body_template = (
+        'Hi %s,<br>'
+        '<br>'
+        'You have %s new message(s) about your Oppia explorations:<br>'
+        '<ul>%s</ul>'
+        'You can view and reply to your messages from your '
+        '<a href="http://www.oppia.org/dashboard">dashboard</a>.'
+        '<br>'
+        'Thanks, and happy collaborating!<br>'
+        '<br>'
+        'Best wishes,<br>'
+        'The Oppia Team<br>'
+        '<br>%s')
+
+    if not feconf.CAN_SEND_EMAILS_TO_USERS:
+        log_new_error('This app cannot send emails to users.')
+        return
+
+    if not feconf.CAN_SEND_FEEDBACK_MESSAGE_EMAILS:
+        log_new_error('This app cannot send feedback message emails to users.')
+        return
+
+    recipient_user_settings = user_services.get_user_settings(recipient_id)
+
+    messages_html = ''
+    for title, messages in feedback_messages.iteritems():
+        for message in messages:
+            messages_html += '<li>%s: %s<br></li>' % (title, message)
```

Comment:
I don't think we should do it. I mean we should send all the feedback because what if we truncate some critical feedback but do not sent it to user due to truncation, user might not check the feedback immediately after receiving email and will be unaware of these feedback unless he opens them in oppia.